src/backend/chat/custom/custom.py

```diff
@@ -302,7 +303,7 @@ def add_files_to_chat_history(
             num_words = min(25, word_count)
             preview = " ".join(file.file_content.split()[:num_words])
 
-            files_message += f"Filename: {file.file_name}\nWord Count: {word_count} Preview: {preview}\n\n"
+            files_message += f"Filename: {file.file_name}\nFile ID: {file.id}\nWord Count: {word_count} Preview: {preview}\n\n"
```

Comment:
Files are indexed into Compass with either convo or agent ID, we can't retrieve a file just with the file name. File search and read tools will take in a tuple (file name, file id) instead.

Reply:
@GangGreenTemperTatum yes